cms/templates/settings.html

```diff
@@ -263,7 +263,7 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                   <label for="course-overview">${_("Course Introduction Video")}</label>
                   <div class="input input-existing">
                     <div class="current current-course-introduction-video">
-                      <iframe width="618" height="350" src="" frameborder="0" allowfullscreen></iframe>
+                      <iframe width="618" height="350" title="Course Introduction Video" src="" frameborder="0" allowfullscreen></iframe>
```

Comment:
@ahsan-ul-haq, we should make this translatable, since it already is above on line 263